sys/include/xtimer.h

```diff
@@ -47,7 +47,7 @@ extern "C" {
  * @note This is a struct in order to make the xtimer API type strict
  */
 typedef struct {
-    uint64_t ticks64;
+    uint64_t ticks64;       /**< Captain Obvious to the rescue! */
```

Comment:
But seriously (sic!), I don't think that this line of code really needs explanation and I was lately a bit worry that RIOT's documentation and comments get a bit too dry lately.